src/languages/en.ts

```diff
@@ -4784,5 +4784,28 @@ export default {
         date: 'Date',
         time: 'Time',
         none: 'None',
+        visibleInLHN: 'Visible in LHN',
+        GBR: 'GBR',
+        RBR: 'RBR',
+        true: 'true',
+        false: 'false',
+        reasonVisibleInLHN: {
+            hasDraftComment: 'Has draft comment',
+            hasGBR: 'Has GBR',
+            pinnedByUser: 'Pinned by user',
+            isNonReimbursedIOU: 'Is non-reimbursed IOU',
+            hasAddWorkspaceRoomErrors: 'Has add workspace room errors',
+            isUnread: 'Is unread (focus mode)',
+            isArchived: 'Is archived (most recent mode)',
+            isSelfDM: 'Is self DM',
+            isFocused: 'Is temporarily focused',
+        },
+        reasonGBR: {
+            hasJoinRequest: 'Has join request (admin room)',
+            isUnreadWithMention: 'Is unread with mention',
+            isWaitingForAssigneeToCompleteAction: 'Is waiting for assignee to complete action',
```

Comment:
```suggestion
            isWaitingForAssigneeToCompleteAction: 'Is waiting for assignee to complete task',
```

Reply:
`action` is more inclusive than `task`

Reply:
Yeah, but isWaitingForAssigneeToCompleteAction only be used in case the user needs to complete the task.

Anyways, It still is fine if we keep 'action'.